server/controller/index.js

```diff
@@ -19,12 +19,14 @@ router.get('/get-languages', languages.get);
 router.get('/get-skills', skills.get);
 router.get('/get-dialcets/:languageId', dialects.get);
 router.get('/get-jobs', jobs.get);
+const voteUp = require('./voteUp');
 
 router.use(checkToken);
 router.get('/questions/:questionId', translation.get);
 router.get('/specialization/:section', questions.get);
 router.get('/specialization', specialization.get);
 router.get('/questions/:questionId', translation.get);
 router.get('/details', userDetails.get);
+router.post('/questions/:questionId/voteUp', voteUp.post);
```

Comment:
urls should be separeated by hyphens rather than camelcase.
```suggestion
router.post('/questions/:questionId/voteUp', vote-up.post);
```